php/ng/bz2.sls

```diff
@@ -0,0 +1,2 @@
+{% set state = 'dba' %}
```

Comment:
I think there's a typo here: it should be bz2

Reply:
You are right, I have fixed it. Although I think #177 is better, supports other distributions